src/routes/Dashboard/Page.svelte

```diff
@@ -1,77 +1,80 @@
 <script lang="ts">
     import COLORS from 'tailwindcss/colors';
-    import { format } from 'date-fns';
     import MdDashboard from 'svelte-icons/md/MdDashboard.svelte';
     import MdSettings from 'svelte-icons/md/MdSettings.svelte';
 
     import Layout from '../../components/Layout.svelte';
+    import LogItem from '../../components/LogItem.svelte';
     import Select from '../../components/Select/Select.svelte';
     import TabGroup from '../../components/Tab/TabGroup.svelte';
     import TabItem from '../../components/Tab/TabItem.svelte';
     import Text from '../../components/Text.svelte';
     import Background from './Background.svelte';
-    import Valve from '../../components/Valve.svelte';
+    import Valve from '../../components/Valve/Main.svelte';
     import type { Option } from '../../components/Select/types.ts';
 
-    import {
-        LOGS,
-        LOG_TYPE_TEXT,
-        LOG_TYPE_COLORS,
-        OPTIONS,
-    } from './constants.ts';
+    import { OPTIONS } from './constants.ts';
     import Display from '../../components/Chart/Display.svelte';
+    import Droplet from '../../assets/droplet.svelte';
+    import DropletCross from '../../assets/droplet-cross.svelte';
 
     let value: Option = OPTIONS[0] || '';
 
-    let active = true;
+    let VALVE_ACTIONS = [
```

Comment:
I believe we don't reassign this, right? The `SCREAMING_CASE` naming convention suggests otherwise.
```suggestion
    const VALVE_ACTIONS = [
```